Code/Tools/FBuild/FBuildCore/Graph/ObjectNode.cpp

```diff
@@ -2389,6 +2389,15 @@ bool ObjectNode::CompileHelper::SpawnCompiler( Job * job,
         // MSVC errors
         if ( objectNode->IsMSVC() )
         {
```

Comment:
The two added conditions seem to duplicate the code above:
```C++
        // If process is manually terminated by a user, consider that a system failure
        // Only do so if there is no error output, as some compilers (like Clang) also
        // use return code 1 for normal compilation failure
        if ( ( result == 0x1 ) && ( stdErr == nullptr ) )
        {
            job->OnSystemError(); // task will be retried on another worker
            return;
        }
}
```
albeit with slightly different conditions. A the time I added the above fix (added ~0.96) I was able to reproduce the behavior and confirm killing the process was handled gracefully. Is it possible we don't need this change, or do you know if the differences in implementation handle some additional case?